benchmarks/dynamo/common.py

```diff
@@ -1890,6 +1868,20 @@ def run(runner, args, original_dir=None):
                     model_name,
                     batch_size=batch_size,
                 )
+                override_model = os.environ.get("override_model", None)
+                if override_model == "linear":
```

Comment:
When debugging some tricky issue like https://github.com/pytorch/xla/issues/4174 , I found it's quite helpful to let torchbench run a simple enough model (rather than a model from torchbench).

This piece of code is not the idea UX. I'm thinking something like:
```
--only=path:<path to model file>;model:<ModelClass>
```
So torchbench will load the model from the path and call ModelClass.get_example_inputs to retrieve the inputs.

Would like to know other's opinion. @jansel  @wconstab

Reply:
that sounds like a useful feature to me. I also use my own ToyModel classes a lot for various bringup work.  I think the way you proposed (with --only) is pretty good.  The hardcoded "linear" thing is probably not worth landing.

Reply:
Here is the separate PR for this: https://github.com/pytorch/pytorch/pull/89028 . @wconstab